packages/workspace/src/utils/runtime-lint-utils.ts

```diff
@@ -70,20 +70,34 @@ export function isRelative(s: string) {
   return s.startsWith('.');
 }
 
-export function isRelativeImportIntoAnotherProject(
+export function getTargetProjectBasedOnRelativeImport(
   imp: string,
   projectPath: string,
   projectGraph: MappedProjectGraph,
-  sourceFilePath: string,
-  sourceProject: ProjectGraphProjectNode
-): boolean {
+  sourceFilePath: string
+) {
   if (!isRelative(imp)) return false;
```

Comment:
The boolean return doesn't seem right based on the function's name. It should be `null` or `undefined`. Please add also return type to the function.